### DISCUSSION TITLE
ZZH on USB device server

### DISCUSSION THREAD

**Discussion by Sineos** [Koenkk/zigbee2mqtt#5670] [Devices]
Could the ZZH stick work with something like the [DIGITUS 1-Port USB 2.0 Multifunction Network Server](https://www.digitus.info/de/produkte/computer-und-office-zubehoer/computer-zubehoer/netzwerk-und-printserver/dn-13020/?PL=en)

Anyone tried or has experience with it?

**Comment by sjorge** [comment]
I have not but https://github.com/tim-brand/zigbee2mqtt-adapter-proxy on rasperry pi and connecting over tcp is working fine for me... well only 1 connection to it at a time ofcourse, but I'd expect that would be the case for the listed device aswel.

**Comment by Sineos** [comment]
Thanks for sharing this. May I ask what the advantage is to proxy from one Pi to the zigbee2mqtt installation? Why not run zigbee2mqtt directly on the Pi to which the adapter is connected?

My initial goal was to prevent YAPIMH - Yet Another Pi In My Home 😉

**Comment by sjorge** [comment]
I just have one Pi which has all my serial devices in my case. I distribute them from there to my VMs, where I have more power.
I can also move the VM between my hardware boxes without issues as it all over tcp.
